python/HeaderService/hslib_salobj.py

```diff
@@ -377,7 +377,14 @@ def collect(self, keys):
             if self.myData[name] is None:
                 self.log.warning(f"Cannot get keyword: {k} from topic: {name}")
             else:
-                self.metadata[k] = getattr(self.myData[name], param)
+                # In case of a long telemetry array we take the first element
```

Comment:
The main telescope will do something similar, but probably have a (roughly) equal number from what we have with the AT... maybe more due to the hexapod etc...